IntegrationTests/UI.IntegrationTests/UITest.cs

```diff
@@ -77,6 +77,29 @@ void HandleApplicationIdle(object sender, EventArgs e)
             }
         }
 
+        public static void RunControl<T>(
+            Func<Form, T> createControl,
+            Func<T, Task> runTestAsync)
+            where T : Control
+        {
+            T control = null;
+            try
+            {
+                RunForm<Form>(
+                    showForm: () =>
+                    {
+                        var form = new Form { Text = $"Test {typeof(T).Name}" };
+                        control = createControl(form);
+                        Application.Run(form);
```

Comment:
💡 Consider adding an assertion here regarding containment, something of the following form (may need to be converted to NUnit syntax):

```csharp
Assert.True(form.Controls.Contains(control));
```

Reply:
done

Reply:
Any chance control could not be in Form.Controls?  Probably want to add an extension method to Form that gets all controls recursively.

Reply:
Can be done if there will be a test case which really needs such a construction.
The test will fail first. So it will be obvious.